src/util.ts

```diff
@@ -88,7 +88,7 @@ export const getNewErrors = (
 }
 
 export const getTotalErrorsCount = (errorMap: Map<string, ErrorInfo>): number =>
-  [...errorMap.values()].reduce((sum, info) => sum + info.count, 0)
+  Array.from(errorMap.values()).reduce((sum, info) => sum + info.count, 0)
```

Comment:
What do you think about adding a comment of caution here for now. Maybe something like:
```suggestion
  // NOTE: Previously, this was written with an array spread, but there was a bug
  // with microbundle that was incorrectly compiling that (see: https://github.com/TimMikeladze/tsc-baseline/issues/21).
  // Until that is resolved or the bundler is switched for this repo, this has been
  // rewritten with Array.from
  Array.from(errorMap.values()).reduce((sum, info) => sum + info.count, 0)
```